### PR TITLE
Adds myself to codeowners

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

yeah I codeown runechat now 😎 
also looked into the file itself and fixed two undefs

## Changelog
:cl:
add: Added myself as codeowner of runechat
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]

File: code/datums/chatmessage.dm [hunk 1]
```diff
@@ -578,3 +578,5 @@
 #undef CHATMESSAGE_CANNOT_HEAR
 #undef CHATMESSAGE_HEAR
 #undef CHATMESSAGE_SHOW_LANGUAGE_ICON
+#undef COLOR_CHAT_EMOTE
+#undef COLOR_PERSON_UNKNOWN
```
Comment (Crossedfall) [inline code/datums/chatmessage.dm]:
```suggestion
```

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Don't mix PRs.

**Comment by Crossedfall** [comment]
I don't really see the need for a codeowner over runechat. What's to be gained from gatekeeping it? It doesn't seem so complex that we need a subject matter expert on it.

**Comment by Crossedfall** [comment]
Closing per my prior statement. I don't see the need for this to be locked behind a single maintainer.

**Pull request closed without merging**